.gitignore

```diff
@@ -29,4 +29,8 @@ bld/
 */dist/
 node_modules
 .vscode
-*.vsix
\ No newline at end of file
+*.vsix
+*.map
+*.js

```

Comment:
I agree with removing these .js files, but we should probably remove them from source as well. The ignore won't ignore already committed files.